examples/text_labels_and_annotations/demo_annotation_box.py

```diff
@@ -3,7 +3,12 @@
 Demo Annotation Box
 ===================
 
+The AnnotationBbox Artist creates an annotation on some location using an
```

Comment:
suggest strike: "on some location".  Vague and doesn't help the sentence....

Reply:
Fixed.